pkg/ops/rawDiskOutput.go

```diff
@@ -44,8 +45,6 @@ func Raw2Azure(source string) (string, error) {
 	var finalSize int64
 	// Calculate the final size in bytes
 	finalSize = ((actualSize + constants.MB - 1) / constants.MB) * constants.MB
-	finalSize -= 512
```

Comment:
So this is what its breaking it? what out because IIRC the checksum is calculated based on the size or something like that? I know I added it somehow for a checksum reason or something like that?

Well, if it works, it works :D